### ISSUE TITLE
Can't distinguish different worktrees of single repo

### ISSUE THREAD

**Issue by koenpunt** [desktop/desktop#5764]
For some of the projects I'm working on I have multiple [`git-worktree`s](https://git-scm.com/docs/git-worktree) set up, but these all show up as the same project name, which makes it quite annoying to work with.

![image](https://user-images.githubusercontent.com/351038/46145835-9c052800-c261-11e8-9a5c-3692d9bd2570.png)

How I think this could be solved is by displaying the branch that is currently checked out for the worktree.

![image](https://user-images.githubusercontent.com/351038/46146153-5ac14800-c262-11e8-8117-169870b73ad2.png)

<sub>Btw, this is not the same as; https://github.com/desktop/desktop/issues/907</sub>

**Comment by outofambit** [comment]
Hi @koenpunt thanks so much for filing this. I think the proposed solution here makes sense as an incremental step but I would love to get some input from design before we jump into implementation.

**Comment by alphajbravo** [comment]
I have a similar issue, except sometimes I'm using the same branch in more than one place (libraries used in different projects, for example), so it would be useful to see where the local is located as well, even just as a tooltip or similar.

**Comment by koenpunt** [comment]
While this was "merged" into #6460, that one was closed, but I think this is still an issue. 

The support for worktrees in general is subpar; I just opened [another issue](https://github.com/desktop/desktop/issues/8043) regarding them.

And also trying to checkout a branch that's already checked out in a different worktree renders an error, instead of something more helpful. 

So I would really appreciate it if my proposal could be reevaluated.
